app/views/coordinators/index.html.erb

```diff
@@ -1,4 +1,11 @@
-<h2>Coordinators</h2>
+<div class="row index-header">
+  <div class="col-md-8 title">
+    <span>Coordinators</span>
+  </div>
+  <div class="col-md-4 text-right">
+    <%= link_to 'New coordinator', new_coordinator_path %>
```

Comment:
do we want to put `new coordinator` at the bottom with `back` or on the right corner? we should make sure we do the same for all